qa/suites/upgrade/nautilus-p2p/nautilus-p2p-parallel/point-to-point-upgrade.yaml

```diff
@@ -33,6 +33,7 @@ overrides:
     - cache pools at or near target size
     - filesystem is degraded
     - OBJECT_MISPLACED
+    - failed to encode map
```

Comment:
let's add a reference to https://tracker.ceph.com/issues/40251 here

Reply:
done, hope it won't break yaml parser :)